test/data/mock-send-state.json

```diff
@@ -130,8 +130,7 @@
     "preferences": {
       "hideZeroBalanceTokens": false,
       "showFiatInTestnets": false,
-      "showTestNetworks": true,
-      "useNativeCurrencyAsPrimaryCurrency": true
```

Comment:
I'm surprised tests didn't fail without you having to mock the new setting.

Reply:
I think lot of tests that use this mock are testing cases not affected by this setting